src/main/java/net/revelc/code/formatter/FormatterMojo.java

```diff
@@ -262,6 +261,22 @@ public class FormatterMojo extends AbstractMojo implements ConfigurationSource {
     @Parameter(defaultValue = "formatter-maven-plugin/ph-css/css.properties", property = "configcssfile", required = true)
     private String configCssFile;
 
+    /**
+     * Whether the formatting cache is skipped.
+     *
+     * @since 2.23.0
+     */
+    @Parameter(defaultValue = "false", property = "formatter.cache.skip")
+    private boolean skipFormattingCache;
+
+    /**
+     * Warn if no files were skipped or changed.
+     *
+     * @since 2.23.0
+     */
+    @Parameter(defaultValue = "false", property = "formatter.warnunchanged")
+    private boolean warnUnchanged;
+
```

Comment:
Since this doesn't actually issue a warning... just puts the results in the warning color in the log, and because this could be confusing when the formatter finds nothing to do (because everything is formatted), then I think this should be removed.

```suggestion
```